var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -19,7 +19,8 @@ class Coreneuron(CMakePackage):
     git      = "https://github.com/BlueBrain/CoreNeuron"
 
     version('develop', branch='master', submodules=True)
-    version('0.21a', commit="bf3c823", preferred=True)
+    version('0.22', tag='0.22', submodules=True, preferred=True)
```

Comment:
BTW, if these are ordered, the `preferred` can be dropped for the latest version. Spack by default will prefer the largest "numerical" version.

Reply:
yes, some recipes have it, not sure why, and in this case came from the coreneuron merge
@All let's keep this in mind for next changes

Reply:
Just wanted to point that out for reference :)

It's a somewhat historic artefact…